### ISSUE TITLE
Unable to set the controller for a subresource

### ISSUE THREAD

**Issue by jeffreymb** [api-platform/core#1828]
I have a plain PHP object (DTO) on which I'm defining an ApiResource as show below. No matter what I do, it seems that the `controller` option is not used. The `path` and `requirements` are working as expected.

```
namespace App\Api\Report;

use ApiPlatform\Core\Annotation\ApiResource;
use ApiPlatform\Core\Annotation\ApiSubresource;
use App\Controller\ApiReportBooksChapters;

/**
 * @ApiResource(
 *     subresourceOperations={
 *          "chapters_get_subresource"={
 *              "path"="report/books/{id}/chapters2.{_format}",
 *              "controller"=ApiReportBooksChapters::class,
 *              "requirements"={
 *                  "id"="[a-z]{3}",
 *              },
 *          },
 *     },
 * )
 */
```

Am I doing something wrong or is this a bug?

**Comment by soyuka** [comment]
https://github.com/api-platform/core/blob/master/src/Bridge/Symfony/Routing/ApiLoader.php#L113 

It's a bug I guess :p. May you patch this?

**Comment by jeffreymb** [comment]
@soyuka PR created.

**Comment by sroze** [comment]
Fixed by #1830

**Comment by soyuka** [comment]
thanks @jeffreymb !
